### ISSUE TITLE
InstanceNorm实现的性能问题

### ISSUE THREAD

**Issue by Windaway** [tensorlayer/tensorlayer#270]
测试中发现InstanceNorm性能较差，看了下实现比较简单，之前用的TF的BN实现去做的，速度还行，刚看了TF contrib的IN实现https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/contrib/layers/python/layers/normalization.py 也是用BN做的，目前IN让我训练时间提高200%，所以觉得有必要优化下了。

**Comment by zsdonghao** [comment]
hi, 能不能贴一下你IN的代码？我尝试包装一下～

**Comment by Windaway** [comment]
我做的是BNlayer做的开train没开updateop做的，而且也没用BN，几乎没法拿来用……

**Comment by Windaway** [comment]
试了下这个应该没错
tl.layers.LambdaLayer(inputs, lambda x: tf.layers.batch_normalization(x,training=True,trainable=False,axis=0),name='instancenorm')

**Comment by Windaway** [comment]
不过不过是在一个instance内部做了……
